### PR TITLE
fix: start timeout after the user started the process

### PR DESCRIPTION
**Pull request by xiduzo**
Prevent the process from terminating early when the user is taking its' time in the form popup

Solves PZ-4513

### CODE REVIEW

**Comment by bas-info-nl** [comment]
Run agenthunt/conventional-commit-checker-action@v2.0.0
Error: Pull request title 
      **************************************************************************************
      fix(batch-process):start timeout after the user started the process
      ************************************************************************************** 
      does not match ^(.+)(?:(([^)s]+)))?: (.+)

File: src/main/app/src/app/shared/batch-progress/batch-process.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,160 @@
+/*
```
Comment (edgarvonk) [inline src/main/app/src/app/shared/batch-progress/batch-process.service.spec.ts:1]: Impressive tests! Just wondering if this is not somewhat overkill for this specific functionality? Maybe not. You know best I guess. :-)

**Review by edgarvonk — APPROVED** [review]
nice!

**Review by bas-info-nl — COMMENTED** [review]
O no, many a copyright comment to update !!

**Pull request merged**